format-sdk/src/main/cpp/CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ include_directories(
 
 set(JAVA_HOME ${THIRDPARTY_DIR}/open_jdk/)
 add_library(jvm SHARED IMPORTED)
-FILE(GLOB_RECURSE LIB_JVM ${JAVA_HOME}/jre/lib/*/libjvm.so)
+FILE(GLOB_RECURSE LIB_JVM ${JAVA_HOME}/lib/*/libjvm.so)
```

Comment:
shall not revert this line back. need to upgrade the JDK version.

Reply:
ignore this comment. the CMakeLists.txt for format-sdk should be kept consistent with the be/CMakeLists.txt